com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/TextureSetGenerator.java

```diff
@@ -310,41 +321,53 @@ public static TextureSetResult calculateLayout(List<Rect> images, List<SpriteGeo
                 .collect(Collectors.toList());
 
         List<Layout> layouts;
-        List<Rect> layoutRects;
-        int layoutWidth;
-        int layoutHeight;
 
         if (useTileGrid) {
             Layout layout = TextureSetLayout.gridLayout(margin, resizedImages, gridSize);
-            layoutWidth = layout.getWidth();
-            layoutHeight = layout.getHeight();
-            layoutRects = layout.getRectangles();
-            layoutRects.sort(Comparator.comparing(o -> o.index));
             layouts = new ArrayList<Layout>();
             layouts.add(layout);
         } else {
-            List<Layout> packedLayouts = TextureSetLayout.packedLayout(margin, resizedImages, rotate, maxPageSizeW, maxPageSizeH);
-            layoutRects = new ArrayList<Rect>();
-
-            int page_index = 0;
-            for (Layout l : packedLayouts) {
-                List<Rect> packedLayoutRects = l.getRectangles();
-                for (Rect r : packedLayoutRects) {
-                    r.page = page_index;
-                }
-                layoutRects.addAll(packedLayoutRects);
-                page_index++;
+            layouts = TextureSetLayout.packedLayout(margin, resizedImages, rotate, maxPageSizeW, maxPageSizeH);
+        }
+
+        // Update the page indices, and remove the extrude border
```

Comment:
Does it remove the extrude border, or is the comment inaccurate?

I ask because it looks like we do this with the `clipBorders` call in `calculateTextureSetResult` below.